src/components/Explore/LocationSelector.tsx

```diff
@@ -36,81 +26,65 @@ const LocationSelector: React.FC<{
-        variant="outlined"
-        disableRipple
-        disableFocusRipple
-        disableTouchRipple
+      <MenuButton
         onClick={onClickButton}
-      >
-        Compare states or counties
-      </Styles.ModalOpenButton>
+        menuLabel="Locations"
+        buttonLabel={regionNames}
+        maxWidth={maxWidth}
+        ariaControlsId={id}
+      />
       {modalOpen && (
-        <Modal
-          open={modalOpen}
-          onClose={closeModal}
-          style={{ touchAction: 'none' }}
-        >
-          <Styles.ModalContainer>
-            <Styles.ModalHeader>
-              <Grid container>
-                <Grid item xs={9}>
-                  <Styles.ModalTitle>
-                    Compare states or counties
-                  </Styles.ModalTitle>
-                </Grid>
-                <Grid item xs container justify="flex-end">
-                  <Styles.DoneButton size="small" onClick={closeModal}>
-                    Done
-                  </Styles.DoneButton>
-                </Grid>
-              </Grid>
-            </Styles.ModalHeader>
-            <Styles.ModalBody>
-              <AutocompleteRegions
-                regions={regions}
-                onChangeRegions={onChangeRegions}
-                selectedRegions={selectedRegions}
-              />
-              {hasMultipleLocations && (
-                <ExploreStyles.NormalizeDataContainer>
-                  <Grid key="legend" item sm xs={12}>
-                    <FormControlLabel
-                      control={
-                        <ExploreStyles.NormalizeCheckbox
-                          checked={normalizeData}
-                          onChange={() => {
-                            setNormalizeData(!normalizeData);
-                          }}
-                          name="normalize data"
-                          disableRipple
-                          id="normalize-data-control"
-                        />
-                      }
-                      label="Normalize Data"
-                    />
-                  </Grid>
-                  <ExploreStyles.NormalizeSubLabel>
-                    Per 100k population
-                  </ExploreStyles.NormalizeSubLabel>
-                </ExploreStyles.NormalizeDataContainer>
-              )}
-            </Styles.ModalBody>
-          </Styles.ModalContainer>
-        </Modal>
+        <>
+          <LockBodyScroll />
+          <Modal
+            open={modalOpen}
+            onClose={closeModal}
+            style={{ touchAction: 'none', display: 'flex' }}
+          >
+            <Styles.ModalContainer>
+              <Styles.ModalHeader>
+                <Styles.ModalTitle>
+                  Compare states, counties, or metro areas
+                </Styles.ModalTitle>
+                <FilledButton
+                  onClick={closeModal}
+                  trackingLabel="Close trends modal"
+                  trackingCategory={EventCategory.EXPLORE}
```

Comment:
We could set `EventCategory.NONE` to skip tracking, but not a big deal if we leave it